tests/cupy_tests/linalg_tests/test_product.py

```diff
@@ -377,6 +377,16 @@ def test_zerodim_kron(self, xp, dtype):
         b = testing.shaped_arange((4, 5), xp, dtype)
         return xp.kron(a, b)
 
+    @testing.numpy_cupy_allclose()
+    def test_kron_accepts_numbers_as_arguments(self, xp):
```

Comment:
You can use `@pytest.mark.parametrize` at the same time with `@testing.numpy_cupy_allclose` here. Did you try it and have some problems?